### PR TITLE
PublicLab scroll-to-top issue#6435

### LINKED ISSUE #6435: Remove "scroll to top" arrow key
Please Note:  **We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more**

### Problem

We can find that the scroll-to-top arrow key also appears when we are at the top of the page.
[here](https://publiclab.org/) 

Screenshot (21)_LI

hence, we should add some javascript scroll function so that it becomes visible only when the user starts to scroll the page

### PR DESCRIPTION
**Pull request by Asiedu13**
Fixes #6435

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: scroll-func.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let scrollBtn = document.querySelector(".back-to-top");
+window.addEventListener("scroll", function(){
+      let position = window.scrollY;
```
Comment (pythongiant) [inline scroll-func.js]: could you format this a bit properly?

**Comment by anthony-zhou** [comment]
@Asiedu13 Looks good! Here are a few changes I would suggest, to run this script on all the pages where the scroll-to-top button appears:

1. To fit the current filename conventions, rename scroll-func.js to scroll_func.js.
2. Move the file scroll_func.js out of the root folder and into app/assets/javascripts.
3. Include the Javascript in the default page layout by adding a `//= require scroll-func.js` statement in [application.js, in the APPLICATION SCRIPTS section](https://github.com/Asiedu13/plots2/blob/7f8103b1e58941a4234d22596e8592775a53513d/app/assets/javascripts/application.js#L61)
4. (optional) Consider using CSS to add an opacity transition to the scroll-to-top button, so that it fades in and fades out.

Thanks!

File: scroll-func.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    if(position < 200){
+      scrollBtn.style.opacity = "0";
+    }else{scrollBtn.style.opacity = "1";}
+})
```
Comment (anthony-zhou) [inline scroll-func.js]: Please add a semicolon at the end of this line, to end the `window.addEventListener` function call. Please see my comment on this PR for more general feedback. Thanks!

**Comment by anthony-zhou** [comment]
@Asiedu13 Looks like PR #6944 is a more recent version of this one, so you can close this pull request.

**Comment by CodeSarthak** [comment]
Hi @Asiedu13 Please close this PR.

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Pull request closed without merging**